packages/suite/src/components/wallet/WalletLayout/WalletLayout.tsx

```diff
@@ -24,6 +25,7 @@ const EmptyHeaderPlaceholder = styled.div`
 
 type WalletLayoutProps = {
     title: ExtendedMessageDescriptor['id'];
+    titleValues?: Record<string, PrimitiveType>;
```

Comment:
```suggestion
    titleValues?: ExtendedMessageDescriptor['values']
```

Reply:
tried this, but ts complains about type compatibility

Reply:
I see, weird